.eslintrc

```diff
@@ -26,7 +26,7 @@
     "no-shadow": 0,
     "no-undef": 2
   },
-  "extends": ["eslint:recommended", "prettier"],
+  "extends": ["eslint:recommended", "airbnb-typescript-prettier"],
```

Comment:
for this stuff, you can create another `.eslintrc` in the folder when you want to use these rules. Otherwise the PR will never be accepted upstream